doc/ref_arch/openstack/chapters/chapter02.md

```diff
@@ -313,15 +317,61 @@ The Platform is assumed to provide configurable alerting and notification capabi
 
-#### 2.2.6.8. Compliance with Standards (source [RM7.9.8](../../../ref_model/chapters/chapter07.md#798-compliance-with-standards))
+#### 2.2.6.9. Open Source Software (source [RM7.9.8](../../../ref_model/chapters/chapter07.md#798-open-source-sotfware))
 
 | Ref # | sub-category | Description |  Traceability |
 |---------|---------------|----------------|------------|
-| sec.std.012 | Standards | The Public Cloud Operator **must**, and the Private Cloud Operator **may** be certified to be compliant with the International Standard on Awareness Engagements (ISAE) 3402 (in the US: SSAE 16); International Standard on Awareness Engagements (ISAE) 3402. US Equivalent: SSAE16. | |
+| sec.oss.001 | Open source code **must** be inspected by tools with various capabilities for static and dynamic code analysis. |  |
+| sec.oss.002 | The CVE(Common Vulnerabilities and Exposures) **must** be used to identify vulnerabilities and their severity rating for open source code part of Cloud Infrastructure and workloads software.  | https://cve.mitre.org/ |
+| sec.oss.003 | A dedicated internal isolated repository separated from the production environment **must** be used to store vetted open source content. |  |
+
+<p align="center"><b>Table 2-13:</b> Reference Model Requirements: Open Source Software Security Requirements</p>
+
+#### 2.2.6.9. IaaC security (source [RM7.9.9](../../../ref_model/chapters/chapter07.md#799-iaac---secure-design-and-architecture-stage-requirements))
+
+**Secure Code Stage Requirements**
+
+| Ref | Requirement | Definition/Note |
+|---|----|---|
+| sec.code.001 | SAST -Static Application Security Testing **must** be applied during Secure Coding stage triggered by Pull, Clone or Comment trigger. | Security testing that analyses application source code for software vulnerabilities and gaps against best practices. Example: open source OWASP range of tools.|
+
+<p align="center"><b>Table 2-14:</b> Reference Model Requirements: IaaC Security Requirements, Secure Code Stage </p>
+
+**Continuous Build, Integration and Testing Stage Requirements**
+
+| Ref | Requirement | Definition/Note |
+|---|----|---|
+| sec.bld.003 | Container and Image Scan **must** be applied during the Continuous Build, Integration and Testing stage triggered by Package trigger. | Example: A push of a container image to a container registry may trigger a vulnerability scan before the image becomes available in the registry.  |
+
+<p align="center"><b>Table 2-15:</b> Reference Model Requirements: IaaC Security Requirements, Continuous Build, Integration and Testing Stage </p>
+
+**Continuous Delivery and Deployment Stage Requirements**
+
+| Ref | Requirement | Definition/Note |
+|---|----|---|
+| sec.del.001 | Image Scan **must** be applied during the Continuous Delivery and Deployment stage triggered by Publish to Artifact and Image Repository trigger. | Example: GitLab uses the open source Clair engine for container scanning.|
```

Comment:
@karinesevilla this is for image scan but the tool is for container

Reply:
@pgoyal01 afaik Clair is a tool for images scanning. We can replace in the example the end of the sentence by 'engine for container image scanning'. Would it works?

Reply:
@karinesevilla In the example, just need to mention "image" scanning.
```suggestion
| sec.del.001 | Image Scan **must** be applied during the Continuous Delivery and Deployment stage triggered by Publish to Artifact and Image Repository trigger. | Example: GitLab uses the open source Clair engine for container image scanning.|
```

Reply:
@pgoyal01 Yes, suggestion added